Cue/app/tabs/library/play/PlayDeckSetupView.js

```diff
@@ -133,6 +210,7 @@ export default class PlayDeckSetupView extends React.Component {
         <ScrollView style={styles.container}>
```

Comment:
No, there's a separate `contentContainerStyle` prop you have to pass in to ScrollView. 

But yeah, something like `contentContainerStyle={{paddingBottom: 40}}`